content/blog/2019/cve-2019-12995/index.md

```diff
@@ -0,0 +1,96 @@
+A bug in Istio’s JWT validation filter causes Istio Proxy to crash in certain cases when the request contains a malformed JWT token. The bug was discovered and reported by a user [on GitHub](https://github.com/istio/istio/issues/15084) on June 23, 2019.
```

Comment:
Istio Proxy -> Envoy, here and elsewhere

Reply:
```suggestion
A bug in Istio’s JWT validation filter causes Envoy to crash in certain cases when the request contains a malformed JWT token. The bug was discovered and reported by a user [on GitHub](https://github.com/istio/istio/issues/15084) on June 23, 2019.
```